src/Nethermind/Nethermind.Mev.Test/Tests/single_gas_0.json

```diff
@@ -17,6 +17,8 @@
       ]
     }
   ],
-  "optimalProfit" : 0,
+  "optimalProfitV1" : 0,
+  "optimalProfitV2_max1bundles" : 0,
```

Comment:
this was meant to have only one optimal profit -> these tests were designed to show how far from optimum particular solutions are so we can design best selectors and profit switchers